cli/schedule.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+from datetime import datetime
+
+import click
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+from indexer.schedule_jobs.aggregates_jobs import aggregates_yesterday_job, parse_crontab
+
+
+@click.command(context_settings=dict(help_option_names=["-h", "--help"]))
+@click.option(
+    "-cn",
```

Comment:
why do you need chain name?

Reply:
Because some configurations depend on the chain, just like the selection of protocol ID in Uniswap v3.
And some table fields have chain names.
But it is true that this will be removed due to the configuration change of the address